packages/scss/src/components/_action-icon.scss

```diff
@@ -36,19 +17,25 @@
 	cursor: pointer;
 
 	& + & {
-		margin-left: .1rem;
+		margin-left: 2px;
 	}
 
-	&:hover {
-		&:not([disabled]) {
+	&:not([disabled]) {
```

Comment:
do we need to add "is-disabled" class too ?

Reply:
Ah, right!